pyproject.toml

```diff
@@ -11,20 +11,68 @@ six = "^1.14"
 pydicom = "^1.4.1"
 coloredlogs = "^14.0"
 pytest-mock = "^3.2.0"
+typer = "^0.3.2"
 
 [tool.poetry.dev-dependencies]
 pytest = "^5.3.5"
 python-dotenv = "^0.12.0"
 black = {version = "^19.10b0", allow-prereleases = true}
 flake8 = "^3.8.3"
 responses = "^0.12.0"
+mkdocs = "^1.1.2"
+mkdocs-material = "^5.5.12"
+mkdocstrings = "^0.13.2"
+typer-cli = "^0.0.10"
+mypy = "^0.782"
+mkdocs-versioning = "^0.3.0"
+Commitizen = "^2.4.1"
+pre-commit = "^2.7.1"
 
 [tool.poetry.scripts]
-bids-postprocess ="xnat_tools.bids_postprocess:run"
-xnat-heudiconv = "xnat_tools.run_heudiconv:run"
-xnat-dicom-export = "xnat_tools.dicom_export:run"
-xnat2bids = "xnat_tools.xnat2bids:run"
+bids-postprocess ="xnat_tools.bids_postprocess:main"
+xnat-heudiconv = "xnat_tools.run_heudiconv:main"
+xnat-dicom-export = "xnat_tools.dicom_export:main"
+xnat2bids = "xnat_tools.xnat2bids:main"
 
-[build-system]
-requires = ["poetry>=0.12"]
+[tool.commitizen]
+version = "0.1.7"
+tag_format = "v$version"
+version_files = [
+    "pyproject.toml:version",
+    "xnat_tools/__init__.py"
+]
+
+[tool.flake8]
+ignore = "E203, E266, E501, W503, F403, F401"
+max-line-length = 89
+max-complexity = 18
+select = "B,C,E,F,W,T4,B9"
+
+[tool.black]
+line-length = 89
+target-version = ['py38']
+exclude = '''
+/(
+    \.git
+  | \.vscode
+  | \.venv
+  | \.mypy_cache
+  | \.pytest_cache
+  | build
+  | dist
+  | site
+)/
```

Comment:
I thought black only worked on `.py` files. Do you need this?

Reply:
Maybe not? I copied from examples

Reply:
Yeah, I just saw that in the examples. I'm surprised, but I guess we should keep it then